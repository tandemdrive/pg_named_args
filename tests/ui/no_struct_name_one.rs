use pg_named_args::query_args;

fn main() {
    let test = 4;
    query_args!("$a, $b", { test });
}
