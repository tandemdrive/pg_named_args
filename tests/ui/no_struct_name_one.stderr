error: expected identifier
 --> tests/ui/no_struct_name_one.rs:5:27
  |
5 |     query_args!("$a, $b", { test });
  |                           ^
